frontend_lib/src/component/Timeline/AutoCompleteTextArea.jsx

```diff
@@ -0,0 +1,253 @@
+import React from 'react'
+import { commonMentionList } from '../../helper'
+import classnames from 'classnames'
+import { translate } from 'react-i18next'
+import PropTypes from 'prop-types'
+import Avatar from '../Avatar/Avatar'
+import AutoCompleteContainer from '../Input/AutoCompleteContainer/AutoCompleteContainer'
+
+export class AutoCompleteTextArea extends React.Component {
```

Comment:
Also, your component naming seems generic but its behavior can only handle the autocompletion of mention. Either name it more precisely or make it more generic